Zyruks/feature/add page planograma

Zyruks opened the pull request:
## Changes Made 🎉

- [x] feat: added hero section with design and content
- [x] feat: added context section with design and content
- [x] style: updated hero section styles for logo and title
- [x] feat: added designs section with images and description
- [x] feat: added process section with journey maps and testing
- [x] refactor: updated class names for contact section
- [x] refactor: modularized sections and updated layout

### Describe Changes

1. **Hero Section**
   - Added a new hero section in `Hero.astro` featuring the main logo, title, description, and three informational cards. The section is styled for both mobile and desktop views with responsive design considerations.
   - Updated styles for the hero section to adjust styles for the main logo and title. The main logo now scales responsively with a max-inline-size, and the title font size uses fluid typography for better scalability across different screen sizes.

2. **Context Section**
   - Added a new context section in `Context.astro` featuring a title, description, image, and informational cards detailing workshop objectives. The section is styled for responsive design, ensuring optimal display on both mobile and desktop devices.

3. **Designs Section**
   - Added a new designs section in `Designs.astro` featuring images and text describing the digitalization of the planogram process. The section is styled for responsive design, ensuring optimal display on both mobile and desktop devices.

4. **Process Section**
   - Added a new process section in `Process.astro` featuring journey maps, testing phases, and analysis descriptions. The section includes images and cards detailing the various stages of the process, with responsive design to ensure optimal display on both mobile and desktop devices.

5. **Contact Section**
   - Refactored the contact section in `contact.astro` to use more generic class names. Updated the container, content, title, description, and actions classes for better readability and maintainability. Styles and layout remain consistent with the previous implementation.

6. **Planograma Page**
   - Refactored `Planograma/index.astro` to modularize the hero, context, process, designs, and contact sections into individual components. Updated the layout to improve readability and maintainability by importing and using these components directly within the main layout.

### Visuals (Optional)

https://github.com/AnaRangel/anarangel.github.io/assets/14036522/e5c37736-a253-4bc7-b7f6-2901b478b445




<!--- Add video or images showcasing the changes or demonstrating the functionality --->

## Checklist ✅

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

Zyruks committed:
feat(planograma): ✨ add planograma page with hero and context sections

Add new Planograma page with a hero section, including logo, title,
description, and methodology, time, and devices cards. Include context
section detailing the challenges and objectives of the planogram project.

- Hero section with logo, title, and description
- Context section with goals and user insights

AnaRangel commented:
Querido  TL Zyruks :· necesito que porfa modifique el tamaño de las cajas contenedoras la la información ya que en el figma las tres cajas tienen el mismo tamaño :3


### implementado
<img width="1152" alt="Captura de pantalla 2024-06-01 a la(s) 10 54 31 p  m" src="https://github.com/AnaRangel/anarangel.github.io/assets/30361612/04cecb94-632c-4340-92e4-cabfbfa6fcab">

### Figma
<img width="1519" alt="Captura de pantalla 2024-06-01 a la(s) 10 54 53 p  m" src="https://github.com/AnaRangel/anarangel.github.io/assets/30361612/c6c50b0a-0c07-42b2-ab9e-40e912014834">

adicional corrige el espacio entre lineas porque esta demasiado apiñado el texto, acomodar el tamaño de la imagen, y el interlineado del tecto de las cards

![image](https://github.com/AnaRangel/anarangel.github.io/assets/30361612/a24a4be9-e96d-4403-ba40-3dc7d36ebe4a)

Zyruks committed:
feat(hero-section): ✨ add hero section with design and content

Added a new hero section in the hero.astro file featuring the main logo,
title, description, and three informational cards. The section is styled
for both mobile and desktop views with responsive design considerations.

Zyruks committed:
feat(context-section): ✨ add context section with design and content

Added a new context section in the Context.astro file featuring a title,
description, image, and informational cards detailing workshop objectives.
The section is styled for responsive design, ensuring optimal display on
both mobile and desktop devices.

Zyruks committed:
style(hero-section): 💄 update hero section styles for logo and title

Updated the hero section in Hero.astro to adjust styles for the main logo
and title. The main logo now scales responsively with a max-inline-size,
and the title font size uses fluid typography for better scalability
across different screen sizes.

Zyruks committed:
chore(assets): 🍱 add assets for planograma page

Zyruks committed:
feat(designs-section): ✨ add designs section with images and description

Added a new designs section in Designs.astro featuring images and text
describing the digitalization of the planogram process. The section is
styled for responsive design, ensuring optimal display on both mobile
and desktop devices.

Zyruks committed:
feat(process-section): ✨ add process section with journey maps and testing

Added a new process section in Process.astro featuring journey maps,
testing phases, and analysis descriptions. The section includes images
and cards detailing the various stages of the process, with responsive
design to ensure optimal display on both mobile and desktop devices.

Zyruks committed:
refactor(contact-section): ♻️ update class names for contact section

Refactored the contact section in contact.astro to use more generic class
names. Updated the container, content, title, description, and actions
classes for better readability and maintainability. Styles and layout
remain consistent with the previous implementation.

Zyruks committed:
chore: 🎨 add barrel to manage _sections imports

Zyruks committed:
refactor(planograma): ♻️ modularize sections and update layout

Refactored Planograma/index.astro to modularize the hero, context, process,
designs, and contact sections into individual components. Updated the layout
to improve readability and maintainability by importing and using these
components directly within the main layout.

Zyruks committed:
feat(pages): ✨ update alt text and comments

Zyruks committed:
chore(pages): ♻️ update planograma url
